plugins/modules/docker_swarm_service.py

```diff
@@ -148,7 +148,7 @@
       - Configure a check that is run to determine whether or not containers for this service are "healthy".
         See the docs for the L(HEALTHCHECK Dockerfile instruction,https://docs.docker.com/engine/reference/builder/#healthcheck)
         for details on how healthchecks work.
-      - "O(healthcheck.interval), O(healthcheck.timeout), and O(healthcheck.start_period) are specified as durations.
+      - "O(healthcheck.interval), O(healthcheck.timeout), O(healthcheck.start_period), and O(healthcheck.start_interval) are specified as durations.
```

Comment:
This and all other changes in this file need to be removed. It is currently not possible to support this option for Docker Swarm services with Docker SDK for Python, since that SDK does not support this new option.

You might want to create a feature request for it in https://github.com/docker/docker-py/issues (or even create a PR there to implement it). Once it's implemented there and a new version with it is released, it can be adde do the docker_swarm_service module.

Reply:
Sorry I've misunderstood with your suggestions on this file, I've removed all changes. I don't think going further on the docker_swarm_service module I don't even know this one before and I've no idea how it works.